code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -229,3 +218,44 @@
 	throwforce = 0
 	breakouttime = 0
 	cuffsound = 'sound/weapons/cablecuff.ogg'
+
+//////////////////////////////
+// MARK: CRAFTING
+//////////////////////////////
+/obj/item/restraints/handcuffs/cable/attackby(obj/item/I, mob/user as mob, params)
+	..()
+	if(istype(I, /obj/item/stack/rods))
+		var/obj/item/stack/rods/R = I
+		if(!R.use(1))
+			to_chat(user, "<span class='warning'>You need one rod to make a wired rod!</span>")
+			return
+
+		var/obj/item/wirerod/W = new /obj/item/wirerod
+		if(!remove_item_from_storage(user))
+			user.unEquip(src)
+		user.put_in_hands(W)
+		to_chat(user, "<span class='notice'>You wrap the cable restraint around the top of the rod.</span>")
+		if(istype(src, /obj/item/restraints/handcuffs/cable/zipties/cyborg))
+			return
```

Comment:
Does this mean that cyborg cuffs are essentially infinite? Wouldn't it maybe be better to just not allow them to make wired rods?

Reply:
I thought it would be amusing to allow them to create wired rods, but I can just make them unable to do it, yes.